framework/base/Component.php

```diff
@@ -475,12 +474,18 @@ public function hasEventHandlers($name)
      */
     public function on($name, $handler, $data = null, $append = true)
```

Comment:
I think that we have to leave this as it is until we define priorities in the App events. Then we can totally replace append + prepend with priority variable as an integer or array